basket/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
```

Comment:
Looks like this file and the `apps.py` work here, but I thought they went in app folders like `base` where the rest of the stuff in the PR went. No?

Reply:
It's here b/c I defined it to be here based on the `basket/apps.py` file pointing to this. It seemed like a good general place to put it that isn't app specific.